EnglishFilter/sections/general_extensions.txt

```diff
@@ -536,7 +536,8 @@ exe.app#%#//scriptlet('prevent-window-open')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/69209
 adsrt.click#%#//scriptlet("prevent-window-open")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/69111
-yoinkexekey.weebly.com#%#//scriptlet("abort-on-property-read", "S9tt")
+! https://github.com/AdguardTeam/AdguardFilters/issues/73839
+mac-torrent-download.net,yoinkexekey.weebly.com#%#//scriptlet("abort-on-property-read", "S9tt")
```

Comment:
what about `mac-torrent-download.net#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "decodeURIComponent")`?

Also, it seems old(in master) scriptlers are not required.

Reply:
This is required, for anti-adblock(blank page) `mac-torrent-download.net#%#//scriptlet("set-constant", "popBlockAlert", "noopFunc")`

Reply:
As I see, these rules are unnecessary:
```
mac-torrent-download.net#%#//scriptlet("abort-current-inline-script", "chAlert")
mac-torrent-download.net#%#AG_onLoad(function() { detectChrome = function() {}; document.cookie = "glx_pp_1_1=1; path=/"; });
mac-torrent-download.net#%#document.cookie = "glx_pp_1_1=1; path=/";
mac-torrent-download.net#%#//scriptlet("set-constant", "glxopen", "noopFunc")
mac-torrent-download.net#%#//scriptlet("set-constant", "wStr", "undefined")
mac-torrent-download.net#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "zfgloaded")
mac-torrent-download.net#%#//scriptlet("abort-on-property-read", "zfgformats")
mac-torrent-download.net#%#Object.defineProperty(HTMLScriptElement.prototype, 'src', {writable:false});
```

Reply:
Yes I kept it.

Reply:
@AdamWr @zloyden @Sergey-Lyapin please also check

Reply:
Isn't it in the filters?

Reply:
>what about `mac-torrent-download.net#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "decodeURIComponent")`?

In this case it breaks whole `String.fromCharCode` function on my end, due to this issue - https://github.com/AdguardTeam/Scriptlets/issues/57#issuecomment-697240649

Reply:
I added @Alex-302 one. It does the same job as @adam's one but in simpler way.

Reply:
I am confused now. @AdamWr @Alex-302 Just add a giant JavaScript Rule which will stop random ad-servers, ads and popups.

Reply:
We can remove `mac-torrent-download.net` from other rules and just add one giant JavaScript rule.

Reply:
@AdamWr On my end it works)

Reply:
On my end it "works" too, but it breaks `String.fromCharCode` function.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/109677363-e1885680-7b79-11eb-9cf8-58e9bcb1affb.png)

</details>

Reply:
I am confused now. @AdamWr @Alex-302 Isn't it possible to add a giant JavaScript Rule which will stop random ad-servers, ads and popups?

Reply:
@AdamWr I mean site is not broken, and everything works.

Reply:
If everything works fine then I don't have anything against adding it:)